### DISCUSSION TITLE
Test Base Property

### DISCUSSION THREAD

**Discussion by franklupo** [microsoft/fluentui-blazor#2435] [Ideas]
I noticed that there are several fixes for missing properties derived from FluentComponentBase e.g. #2420 . 
Wouldn't it be possible to create a default test that checks for them? Do we always want Id, AdditionalAttributes, Class, Style or am I wrong?

**Comment by vnbaaij** [comment]
Up till now, we found we did not always want or need all of the Id, AdditionalAttributes, Class or Style attributes/parameters on components. For instance, for `FluentBreadcrumb` we do not pass an id to the web component (that was the first I could find). 

Think we need to determine what we need on a case-by-case base ad not make these required (and add when needed such as with Accordion recently)

**Reply by franklupo** [reply]
IMO  `Id` and `AdditionalAttributes` should always be there. If they are not there it is an error

**Reply by vnbaaij** [reply]
`Id` we can think about. `AdditionalAttributes` might bring in a small performance hit so don't think we will always want to have that one.
